openstudio-standards.gemspec

```diff
@@ -51,7 +51,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rubyXL', '~> 3.4'
   spec.add_development_dependency 'simplecov', '0.22.0'
   spec.add_development_dependency 'yard', '~> 0.9'
-  spec.add_runtime_dependency 'tbd'
+  spec.add_runtime_dependency 'tbd', '~> 3.4.4'
```

Comment:
The _autozone_ sizing runs rely on an older OpenStudio linking to a much older _OSut_ gem:

`osut (0.2.8) ':/ruby/2.7.0/gems/osut-0.2.8'`

Updating the _gemspec_ and _Gemfile.lock_ files to set the minimum _TBD_ & _OSut_ gem versions. This is not fixing the observed issue, yet keeping it here as a reminder.